cloudant-client/src/main/java/com/cloudant/client/org/lightcouch/CouchDbClient.java

```diff
@@ -277,6 +279,46 @@ public Replicator replicator() {
         return new Replicator(this);
     }
 
+
+    /**
+     * Lists replication jobs. Includes replications created via /_replicate endpoint as well as
+     * those created from replication documents. Does not include replications which have
+     * completed or have failed to start because replication documents were malformed. Each job
+     * description will include source and target information, replication id, a history of
+     * recent event, and a few other things.
+     *
+     * @return List of replication jobs
```

Comment:
Same doc comments as above, given this class is an internal thing it probably doesn't need this duplication anyway.